packages/retryable-monitor/index.ts

```diff
@@ -265,8 +265,8 @@ const processChildChain = async (
 
     const childChainTicketReport = {
       id: retryableMessage.retryableCreationId,
-      retryTxHash:
-        (await retryableMessage.getAutoRedeemAttempt())?.transactionHash || '',
```

Comment:
This `|| ''` was the issue since we were expecting `retryTxHash` to be `null / undefined` to show "No auto-redeem attempt found" message, leading to empty tx hash link.